Makefile

```diff
@@ -284,7 +269,7 @@ run-kube-proxy:
 	-docker rm -f calico-kube-proxy
 	docker run --name calico-kube-proxy -d --net=host --privileged gcr.io/google_containers/hyperkube:v$(K8S_VERSION) /hyperkube proxy --master=http://127.0.0.1:8080 --v=2
 
-ci: clean static-checks-containerized test-containerized docker-image
+ci: clean static-checks test-containerized docker-image
```

Comment:
not totally related, but can we rename the `docker-image` target to something more relevant?